features/add_a_book.feature

```diff
@@ -0,0 +1,50 @@
+Feature: Add book to a library
+
+
+    Scenario: Admin adds new book to the library giving correct data
+
+        Given Admin is logged in
+
+        And admin is on the „library” page
```

Comment:
Trzymaj się jednej konwencji. Po Given When Then And albo używaj zawsze dużych liter albo małych :)